### PR TITLE
Merged dependabot pull requests, fixed gosec linter

### PR DESCRIPTION
**Pull request by jvandellen**
Merged dependabot pull requests, fixed gosec linter issues during build

### CODE REVIEW

**Review by sgsullivan — APPROVED** [review]
lgtm :robot:

Ideally going forward to improve this project more:

1. Add at least basic "smoke test" unit/integration tests.
2. Wrap 1 in a github action check, required via branch protection rule on main branch.
3. Consider updating current semi-manual release process to automatically publish a github release when a commit enters main branch. Bonus points if said release includes common usable artifacts a customer would want (like an exe, rpm, deb, mac executable, etc).

File: go.mod [hunk 1]
```diff
@@ -3,26 +3,15 @@ module github.com/liquidweb/liquidweb-cli
 go 1.13
 
 require (
-	github.com/c-bata/go-prompt v0.2.5
-	github.com/fsnotify/fsnotify v1.4.9 // indirect
+	github.com/c-bata/go-prompt v0.2.6
 	github.com/google/go-github/v32 v32.1.0
 	github.com/k0kubun/go-ansi v0.0.0-20180517002512-3bf9e2903213
 	github.com/liquidweb/go-lwApi v0.0.5
```
Comment (jakdept) [inline go.mod:9]: we need a card for this line. I'll look at it next wek if you want for you.

**Review by jakdept — APPROVED** [review]
lgtm

**Comment by jakdept** [comment]
yeah @sgsullivan we're going to be putting more time into this, and I've always been obsessive about testing and stuff like github actions and codeowners. Expect to see more over time. Currently I'm more focused on terraform, and there's shortly going to be a massive change to `liquidweb-go` (that'll be backwards compat) but our plan is to put work into this and more.

**Pull request merged**